labshare-client/src/main.js

```diff
@@ -1,23 +1,26 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
-import router from './router.js'
-import store from './store'
-import App from './App.vue'
-import './http'
-import profile_load_mixin from './mixins/profile_fetch'
-import { BootstrapVue, IconsPlugin } from 'bootstrap-vue'
+import Vue from "vue";
```

Comment:
My VSCode is apparently set to prefer double-quotes and automatically put in semicolons. I can can change these settings if that's an issue.

Reply:
On the other hand, we should probably just use Prettier to automatically flatten these issues between different workstations